README.md

```diff
@@ -91,6 +91,11 @@ make install
 ```
 
 
+## Contributing
+
+If you would like to contribute to the OpenSCAP, either fix some issue or add a new feature, please check our [contribution guide](docs/contribute/contribute.adoc). In case you would like to discuss anything or ask questions you can either join our FreeNode IRC channel called **#openscap** or our [mailing list](https://www.redhat.com/mailman/listinfo/open-scap-list).
```

Comment:
the `either.. or..` wording sounds weird here. I am not a native speaker though

Reply:
I'd suggest:

> We welcome all contributions to the OpenSCAP project. If you would like to contribute, either by fixing existing issues or adding new features, please check out our [contribution guide](docs/contribute/contribute.adoc) for help getting started. If you would like to discuss anything, ask questions, or if you need additional help getting started, you can either send a message to our FreeNode IRC channel, **#openscap**, or to our [mailing list](https://www.redhat.com/mailman/listinfo/open-scap-list).

Reply:
Thanks, I will use the suggested text.